cygnus-ngsi/conf/agent_ngsi.conf.template

```diff
@@ -17,7 +17,7 @@
 #=============================================
 # To be put in APACHE_FLUME_HOME/conf/agent.conf
 #
-# General configuration template explaining how to setup a sink of each of the available types (HDFS, CKAN, MySQL, PostgreSQL).
+# General configuration template explaining how to setup a sink of each of the available types (HDFS, CKAN, MySQL, PostgreSQL, Mongo, FIWARE Comet, Kafka, DynamoDB, CartoDB).
```

Comment:
Fixed in 75ecb11 (Following this issue i've replaced more references to FIWARE Comet with STH Comet ^^ )